data/helpers.d/psql

```diff
@@ -0,0 +1,150 @@
+# Create a master password and set up global settings
+# Please always call this script in install and restore scripts
+#
+# usage: ynh_psql_test_if_first_run
+ynh_psql_test_if_first_run() {
+	if [ -f /etc/yunohost/psql ];
+	then
+		echo "PostgreSQL is already installed, no need to create master password"
+	else
+		pgsql=$(ynh_string_random)
```

Comment:
What is $pgsql ?
Isn't this variable should be locale ?

Reply:
Yes it should.

Reply:
It's used to store the pgsql password for `postgres` user. A comment must be added.